### PR TITLE
Argument changed back to right key

### PR DESCRIPTION
**Pull request by jontours**
Addresses filter bug(s) from: https://github.com/18F/openFEC/issues/2157.

### CODE REVIEW

File: webservices/resources/sched_e.py [hunk 1]
```diff
@@ -109,11 +109,15 @@ class ScheduleEEfileView(views.ApiResource):
         ('committee_id', models.ScheduleEEfile.committee_id),
         ('candidate_id', models.ScheduleEEfile.candidate_id),
         ('support_oppose_indicator', models.ScheduleEEfile.support_oppose_indicator),
-        ('candidate_name', models.ScheduleEEfile.candidate_name),
+        #('candidate_name', models.ScheduleEEfile.candidate_name),
```
Comment (noahmanger) [inline webservices/resources/sched_e.py]: Is this going to remove the candidate_name filter?

File: webservices/resources/sched_e.py [hunk 2]
```diff
@@ -109,11 +109,15 @@ class ScheduleEEfileView(views.ApiResource):
...
     ]
 
     filter_range_fields = [
-        (('min_date', 'max_date'), models.ScheduleEEfile.expenditure_date),
+        (('min_expenditure_date', 'max_expenditure_date'), models.ScheduleEEfile.expenditure_date),
+    ]
+
+    filter_fulltext_fields = [
+        ('candidate_name', models.ScheduleEEfile.candidate_name),
```
Comment (jontours) [inline webservices/resources/sched_e.py]: @noahmanger it moved to here.

Comment (noahmanger) [inline webservices/resources/sched_e.py]: Ah ha. Shows me for reviewing code I'm not familiar with.

**Comment by ccostino** [comment]
Thanks, @jontours!

**Pull request merged**